extensions/amp-samba-player/0.1/validator-amp-samba-player.protoascii

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-samba-player
```

Comment:
We recently simplified the way to validate extension scripts into a new format. The new format for this tag would look like this:
```
tags: {  # amp-samba-player
  html_format: AMP
  tag_name: "SCRIPT"
  satisfies: "amp-samba-player extension .js script"
  requires: "amp-samba-player"
  extension_spec: {
    name: "amp-samba-player"
    allowed_versions: "0.1"
    allowed_versions: "latest"
  }
  attr_lists: "common-extension-attrs"
}
```